pxd.c

```diff
@@ -261,8 +265,8 @@ static void print_io_flusher_state(unsigned int new_flags,
 
 static int is_io_flusher_supported(void)
 {
-	#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0) && LINUX_VERSION_CODE <= KERNEL_VERSION(5,8,0)
-		#if defined(PF_MEMALLOC_NOIO) && defined(PF_LESS_THROTTLE)
+  	#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0) && defined(PF_MEMALLOC_NOIO)
+                #if defined(PF_LESS_THROTTLE) || defined(PF_LOCAL_THROTTLE)
```

Comment:
the above macro seems incorrect, checking same entity between OR?

Reply:
Checking the same entity?   I am sorry I don't understand what you mean?

Reply:
PF_LESS_THROTTLE is a generic kernel #define which is defined include/linux/sched.h
 PF_LESS_THROTTLE and PF_LOCAL_THROTTLE have different meaning and implementation, I don't think we should be assigning PF_LESS_THROTTLE = PF_LOCAL_THROTTLE

Reply:
We can change the implementation such that all the new code related to IO_FLUSHER is compile time enabled based on 
#ifdef PF_IO_FLUSHER
#endif

This can be taken up as a new PR